app/src/main/res/layout/marvel_hero_row.xml

```diff
@@ -4,32 +4,27 @@
     android:layout_height="250dp">
 
     <ImageView
-        android:id="@+id/im_hero_image"
+        android:id="@+id/heroImage"
```

Comment:
I prefer the imHeroImage or tvEmptyList annotation because we know the type of the element